packages/core/src/customizations/components/fragments/pdp.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@faststore/graphql-utils'
+
+export const fragment = gql`
+  fragment pdp on Query {
+    product(locator: [{ key: "slug", value: $slug }]) {
```

Comment:
what about change it to use the locator
```suggestion
    product(locator: $locator) {
```
so that we could also add the `...pdp` in the useProduct hook?
references: 
- https://github.com/vtex/faststore/pull/1927/commits/9dab11831787ea5cdb8a586e49da5b5bbf25e6b2
- https://github.com/vtex/faststore/pull/1927/commits/37d9a7d924a5d0d43ed974a1cadabf86b6cd1816

Reply:
I agree, done!